components/collator/tests/tests.rs

```diff
@@ -1404,6 +1404,20 @@ fn test_conformance_non_ignorable() {
     }
 }
 
+#[test]
+fn test_case_level() {
+    let mut options = CollatorOptions::new();
+    options.strength = Some(Strength::Primary);
+    options.case_level = Some(CaseLevel::On);
+    let collator_with_case =
+        Collator::try_new_unstable(&icu_testdata::unstable(), &Default::default(), options)
+            .unwrap();
+    assert_ne!(
+        collator_with_case.compare("aA", "Aa"),
+        core::cmp::Ordering::Equal
```

Comment:
```suggestion
        core::cmp::Ordering::Less
```

Reply:
This is `assert_ne`.

Reply:
Incorporated the suggestion in the fixup.